consensus/types/src/config_and_preset.rs

```diff
@@ -72,7 +74,7 @@ impl ConfigAndPreset {
                 extra_fields,
             })
         } else if spec.electra_fork_epoch.is_some()
-            || fork_name.is_none()
```

Comment:
This was unreachable before, as it had been copy-pasta'd to the Fulu case. Using `latest_stable_fork` prevents this kind of issue.